src/scss/style.scss

```diff
@@ -0,0 +1,92 @@
+.module {
+    color: #625F5F;
+    border-radius: 2px;
+    padding: 15px;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #C8C4C3;
+    min-height: 100vh;
+
+    .wrapper {
+        display: flex;
+        border: 1px solid #C8C4C3;
+        border-radius: 10px;
+        justify-content: center;
+        flex-direction: column;
+        -webkit-box-shadow: 6px 9px 5px -4px rgba(0,0,0,0.44);
+        -moz-box-shadow: 6px 9px 5px -4px rgba(0,0,0,0.44);
+        box-shadow: 6px 9px 5px -4px rgba(0,0,0,0.44);
+    
+        h6 {
+            border-bottom: 1px solid #C8C4C3;
+            margin: 0;
+            padding: 15px;
+            text-transform: uppercase;
+        }
+    
+        .content {
+            display: flex;
+        }
+    
+        
+        .all-added-nums-sum {
+            border-right: 1px solid #C8C4C3;
+            padding: 10px;
+
+            .added-nums-container {
+                height: 60vh;
```

Comment:
In this layout setup, the size of the whole panel is defined from its content.  It should be the other way around.
You should have a defined (somehow) size of the panel and the content should take its space.